core/tests/test_utils.py

```diff
@@ -1731,6 +1732,52 @@ def get_static_asset_url(self, asset_suffix):
         """
         return '/assets%s%s' % (utils.get_asset_dir_prefix(), asset_suffix)
 
+    @contextlib.contextmanager
+    def mock_datetime_utcnow(self, mocked_datetime):
+        """Mocks response from datetime.datetime.utcnow method
+
+        Args:
+            mocked_datetime: datetime.datetime.
+                The datetime which will be used instead of current
+
+        Yields:
+            nothing
+
+        Example usage:
```

Comment:
Put this below the top line of the docstring (after one newline). Our format is typically "one line docstring, more details, Args/Returns/Raises/Yields sections".

Reply:
Fixed